content/agent/autodiscovery.md

```diff
@@ -488,6 +488,8 @@ services:
 {{% /tab %}}
 {{< /tabs >}}
 
+**Note**: Some supported integrations require additional steps for Autodiscovery to work: [Ceph][3], [Varnish][4], [Postfix][5], [Cassandra Nodetools][6], and [Gunicorn][7]. Contact [Datadog support][8] for assistance.
+
```

Comment:
@ruthnaebeck any specifics you can share here?

Reply:
@irabinovitch - These are best effort support. There have been some work-a-rounds implemented for customers in the past with the assistance of the containers team but it's on a case by case basis. It's best for a customer to reach out to support for their case.

Reply:
Let's get this correctly documented. Will reach out directly with more info.